app/models/preview_card.rb

```diff
@@ -34,7 +34,7 @@ class PreviewCard < ApplicationRecord
 
   has_and_belongs_to_many :statuses
 
-  has_attached_file :image, styles: { original: { geometry: '400x400>', file_geometry_parser: FastGeometryParser } }, convert_options: { all: '-quality 80 -strip' }
+  has_attached_file :image, styles: { original: { geometry: '400x400>', format: 'jpg', file_geometry_parser: FastGeometryParser } }, convert_options: { all: '-quality 80 -strip' }
```

Comment:
Would it make sense to do this similar to account avatars with a lambda proc based on the file?

```
has_attached_file :image, styles: ->(f) { image_styles(f) }, convert_options: { all: '-quality 80 -strip' }

def image_styles(file)
  styles = { original: { geometry: '400x400>', file_geometry_parser: FastGeometryParser } }
  styles.original.format = "png"  if file.content_type == 'image/gif'
  styles
end
```